app/views/questions/_question.html.erb

```diff
@@ -1,6 +1,9 @@
   <%= link_to question_path(question), class:"list-group-item list-group-item-action list-group-item-secondary" do %>
   <div class= "d-flex justify-content-between bd-highlight">
     <Strong class=""><%= question.title %></Strong>
+    <div class= "d-flex flex-column flex-shrink-1">
+      <div><%="#{question.answers.count} Answers"%></div>
```

Comment:
difference between count and size?

Reply:
count calls the query where as size checks id it is preloaded size then takes length of it otherwise calls count